lte/gateway/c/oai/tasks/mme_app/mme_app_ip_imsi.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include "log.h"
+#include "common_defs.h"
+#include "common_types.h"
+void initialize_ipv4_map(void);
+int mme_app_insert_ue_ipv4_addr(uint32_t ipv4_addr, imsi64_t imsi64);
+int mme_app_get_imsi_from_ipv4(uint32_t ipv4_addr, imsi64_t** imsi_list);
```

Comment:
why do you need double pointer but not simply use `imsi64_t* imsi_list`?

Reply:
Within this mme_app_get_imsi_from_ipv4(), I will be able to get the number of IMSIs and list of IMSIs; So I allocated memory with obtained number of IMSIs and filled with IMSIs matched for specific ue ip address.
Since I am allocating memory within this fun() I used the double pointer.
The caller of this fun() needs list of IMSIs, so it sends pointer, in which allocated memory needs to be allocated.
This fun() returns number of IMSIs matched for specific ue ip address.